scripts/install-multi-user.sh

```diff
@@ -627,14 +730,27 @@ configure_shell_profile() {
                         tee -a "$profile_target"
         fi
     done
+    # TODO: should we suggest '. $PROFILE_NIX_FILE'? It would get them on
+    # their way less disruptively, but a counter-argument is that they won't
+    # immediately notice if something didn't get set up right?
+    reminder "Nix won't work in active shell sessions until you restart them."
+}
+
+cert_in_store() {
+    # in a subshell
+    # - change into the cert-file dir
+    # - get the phyiscal pwd
+    # and test if this path is in the Nix store
+    [[ "$(cd -- "$(dirname "$NIX_SSL_CERT_FILE")" && exec pwd -P)" == "$NIX_ROOT/store/"* ]]
 }
 
 setup_default_profile() {
-    _sudo "to installing a bootstrapping Nix in to the default Profile" \
+    task "Setting up the default profile"
+    _sudo "to install a bootstrapping Nix in to the default profile" \
           HOME="$ROOT_HOME" "$NIX_INSTALLED_NIX/bin/nix-env" -i "$NIX_INSTALLED_NIX"
 
-    if [ -z "${NIX_SSL_CERT_FILE:-}" ] || ! [ -f "${NIX_SSL_CERT_FILE:-}" ]; then
-        _sudo "to installing a bootstrapping SSL certificate just for Nix in to the default Profile" \
+    if [ -z "${NIX_SSL_CERT_FILE:-}" ] || ! [ -f "${NIX_SSL_CERT_FILE:-}" ] || cert_in_store; then
```

Comment:
```suggestion
    if [ -z "${NIX_SSL_CERT_FILE:-}" || ! -f "${NIX_SSL_CERT_FILE:-}" ] || cert_in_store; then
```

Reply:
```console
$ [ -z "${NIX_SSL_CERT_FILE:-}" ] || ! [ -f "${NIX_SSL_CERT_FILE:-}" ]; echo $?
1
$ [ -z "${NIX_SSL_CERT_FILE:-}" || ! -f "${NIX_SSL_CERT_FILE:-}" ]
-bash: [: missing `]'
-bash: -f: command not found
```

This would work with `[[ ... ]]`, but I'm already squeamish about this change (and I don't intend to fold in a general `[` -> `[[` conversion, here--I agree with doing one for the small runtime improvement, but I'm also squeamish about the sheer size of the changeset already; conditional changes like this will be fairly easy to review as a focused commit).